gensim/models/ldamodel.py

```diff
@@ -579,6 +579,15 @@ def update(self, corpus, chunksize=None, decay=None, offset=None,
         if gamma_threshold is None:
             gamma_threshold = self.gamma_threshold
 
+        if isinstance(corpus, list):
+            logger.warning(
```

Comment:
What does this warning message look like when actually emitted?

The spacing around line breaks and punctuation doesn't seem right.